types/node/v8/index.d.ts

```diff
@@ -5169,9 +5169,9 @@ declare module "crypto" {
         update(data: string, input_encoding: HexBase64BinaryEncoding, output_encoding: Utf8AsciiBinaryEncoding): string;
         final(): Buffer;
         final(output_encoding: string): string;
-        setAutoPadding(auto_padding?: boolean): void;
+        setAutoPadding(auto_padding?: boolean): Decipher;
         setAuthTag(tag: Buffer): void;
```

Comment:
I have seen that `setAuthTag()` also returns `this` (see https://nodejs.org/api/crypto.html#crypto_decipher_setauthtag_buffer) . Maybe you want to fix this also.